apps/api/src/domain/models/audio-item/entities/audio-item-composite-identifier.ts

```diff
@@ -1,17 +1,56 @@
-import { NonEmptyString, UUID } from '@coscrad/data-types';
+import { ExternalEnum, NonEmptyString, UUID } from '@coscrad/data-types';
 import { AggregateId } from '../../../types/AggregateId';
 import { AggregateType } from '../../../types/AggregateType';
+import { ResourceType } from '../../../types/ResourceType';
+
+const audiovisualResourceTypes = [ResourceType.video, ResourceType.audioItem] as const;
+
+export type AudiovisualResourceType = typeof audiovisualResourceTypes[number];
 
 export class AudioItemCompositeIdentifier {
     @NonEmptyString({
         label: 'type',
-        description: 'transcript',
+        description: 'audio item',
     })
     type = AggregateType.audioItem;
 
     @UUID({
         label: 'ID',
-        description: 'the transcript ID (generated)',
+        description: 'the audio item ID (generated)',
+    })
+    id: AggregateId;
+}
+
+export class AudioVisualCompositeIdentifier {
+    @ExternalEnum(
+        {
+            labelsAndValues: [
+                {
+                    label: 'video',
+                    value: ResourceType.video,
+                },
+                {
+                    label: 'audio item',
```

Comment:
Just wondering why it needs to be `audio item` rather than just `audio` as in `video`?